addons/pf-maint.pl

```diff
@@ -40,6 +40,8 @@ =head1 SYNOPSIS
 use IO::Handle;
 use Term::ReadKey;
 use IO::Interactive qw(is_interactive);
+# use CONNECT method to deal with proxy, see #3425
+$ENV{PERL_NET_HTTPS_SSL_SOCKET_CLASS} = "Net::SSL";
```

Comment:
Make sure perl-Crypt-SSLeay is a dependency of PacketFence.

Reply:
On a CentOS 7.6 with PF installed:
```
# yum list installed|grep perl|grep -i ssl
[..]
perl-Crypt-SSLeay.x86_64                   0.64-5.el7               @base       
perl-IO-Socket-SSL.noarch                  2.049-1.1                @packetfence
[..]
perl-Net-SSLeay.x86_64                     1.55-6.el7               @base 
```

Reply:
My code is wrong, with only `$ENV{PERL_NET_HTTPS_SSL_SOCKET_CLASS} = "Net::SSL";`

We need to add: `$ENV{PERL_LWP_SSL_VERIFY_HOSTNAME} = 0;` 

which is not a good idea for me, see [verify_hostname option](https://metacpan.org/pod/LWP::UserAgent#verify_hostname-=%3E-$bool) for details.

My two proposals:
- add documentation to explain how to use a proxy on CentOS 7 (with the `$ENV` variables)
- package `perl-libwww-perl` in PF repo for CentOS 7 at version 6.08 (same version as Debian)

Reply:
If the certificate match the fqdn then you don't need to set $ENV{PERL_LWP_SSL_VERIFY_HOSTNAME} = 0;
For the perl-Crypt-SSLeay you need to add it in the spec file and in the debian control file.

Reply:
> If the certificate match the fqdn then you don't need to set $ENV{PERL_LWP_SSL_VERIFY_HOSTNAME} = 0;

Didn't work with api.github.com, maybe due to wildcard.